.travis/build-windows.sh

```diff
@@ -6,20 +6,29 @@ export PATH=$MSBUILD_PATH:$PATH
 mkdir build
 cd build
 
-if [ "$1" == "release" ]; then
+if [ "$1" = "release" ]; then
+    if [ "$2" = "openmp" ]; then
+        ENABLE_OPENMP="ON"
+    else
+        ENABLE_OPENMP="OFF"
+    fi
     BUILD_TYPE=RelWithDebInfo
     cmake \
-        -G "Visual Studio 15 2017 Win64" \
+        -G "Visual Studio 15 2017" \
         -DCMAKE_BUILD_TYPE="${BUILD_TYPE}" .. \
-        -DENABLE_OPENMP=ON \
-        -DENABLE_LTO=ON
+        -DENABLE_OPENMP="${ENABLE_OPENMP}" \
+        -DENABLE_LTO=ON \
+        -DCMAKE_GENERATOR_PLATFORM="Win32"
 else
     BUILD_TYPE=Debug
     cmake \
-        -G "Visual Studio 15 2017 Win64" \
-        -DCMAKE_BUILD_TYPE="${BUILD_TYPE}" ..
+        -G "Visual Studio 15 2017" \
+        -DCMAKE_BUILD_TYPE="${BUILD_TYPE}" \
+        -DCMAKE_GENERATOR_PLATFORM="Win32" ..
 fi
 
 cmake --build . --config "${BUILD_TYPE}" -- -maxcpucount
 
 bin/$BUILD_TYPE/solvespace-testsuite.exe
+
+mv bin/$BUILD_TYPE/solvespace.exe bin/$BUILD_TYPE/solvespace-openmp.exe
```

Comment:
Should this be done only `if [ "$2" = "openmp" ];` ?

Reply:
Good catch!